SoftwareCo/SoftwareCo/Tracker/managers/HashManager.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Sodium;
+
+namespace SoftwareCo
+{
+    public class HashManager
+    {
+        public static String HashValue(string value, string dataType)
+        {
+            if (!CacheManager.HasJwt() || string.IsNullOrEmpty(value))
+            {
+                return "";
+            }
+
+            try
+            {
+                byte[] key = GenericHash.GenerateKey(); // 64 byte key
+                string hashedValue = Utilities.BinaryToHex(GenericHash.Hash(value, key, 128));
```

Comment:
I'm pretty sure that if we use keys, our hashes for the same value become random.    Did you check that running this twice on the same original value will produce the same hashed value?

Reply:
Yeah. Key would make it different on every run and different for users on the same team.